fcrepo-kernel-impl/src/main/resources/sql/mysql-containment.sql

```diff
@@ -3,10 +3,11 @@
 
 -- Holds the ID and its parent.
 CREATE TABLE IF NOT EXISTS containment (
-    fedora_id  varchar(503) NOT NULL,
+    fedora_id varchar(503) NOT NULL,
     parent varchar(503) NOT NULL,
     start_time timestamp NOT NULL,
-    end_time timestamp NULL
+    end_time timestamp NULL,
+    UNIQUE KEY (fedora_id)
```

Comment:
You can just add the `PRIMARY KEY` to the `fedora_id` line, I'm not sure why it disappeared but clearly I made a mistake.

Reply:
Yeah, I can change that. I originally wrote it that way because I was browsing some discussion about varchar primary keys, which a lot of folks advise against for performance reasons. In this case, the key difference between using `PRIMARY KEY` vs `UNIQUE KEY` is that the former creates a clustered index and the later creates a non-clustered index. Because we will not be adding keys sequentially, this is less ideal for clustered indexes.

That said, I don't mind changing it. All of the other databases are currently defined with `PRIMARY KEY`, lookup should still be faster with the clustered index, and it's unclear how much of a difference it even makes.

Reply:
I agree that having someone with more indepth DB experience have a look at our schemas (<wink> <wink> @birkland) would be helpful. If you want to leave this as is, I'm fine. Just don't want people to think there is a specific reason that MySQL is different.